srvutil/metrics_test.go

```diff
@@ -75,7 +75,7 @@ func TestRequestMetricsMiddleware(t *testing.T) {
 	assert.Equal(t, "hello world", string(body))
 
 	assert.NotNil(t, recordedTags, "should have recorded a %s tag", metrics.HTTPRequest.Name)
-	assert.Equal(t, []string{"route:/hello/@name", "statusClass:2xx", "statusCode:200", "success:true"}, recordedTags)
```

Comment:
I thought this success tag was doing something important then realize it's pointless due to the callback always returning nill